src/Core/Field.cpp

```diff
@@ -146,7 +147,7 @@ inline Field getBinaryValue(UInt8 type, ReadBuffer & buf)
         case Field::Types::CustomType:
             return Field();
     }
-    UNREACHABLE();
+    throw Exception(ErrorCodes::INCORRECT_DATA, "Unknown field type {}", std::to_string(type));
```

Comment:
You don't need `std::to_string` right?

Reply:
For odd reasons, fmtlib produces a gigantic error message for `UInt8`, other integer types work. Guess it can't concatenate strings and chars ...

Reply:
Odd. We probably missed some include somewhere, but it's ok to leave this as is and backport it.